src/NUnitFramework/framework/Api/FrameworkController.cs

```diff
@@ -105,8 +98,8 @@ public FrameworkController(Assembly assembly, string idPrefix, IDictionary setti
         /// <param name="builderType">The Type of the test builder</param>
         public FrameworkController(string assemblyNameOrPath, string idPrefix, IDictionary settings, string runnerType, string builderType)
         {
-            Builder = (ITestAssemblyBuilder)Activator.CreateInstance(Type.GetType(builderType));
-            Runner = (ITestAssemblyRunner)Activator.CreateInstance(Type.GetType(runnerType), new object[] { Builder });
+            Builder = (ITestAssemblyBuilder)Reflect.Construct(Type.GetType(builderType));
```

Comment:
Yes, I thought of doing  that but decided I didn't want to make a global change right now.